import-service/src/functions/import-file-parser/handler.ts

```diff
@@ -0,0 +1,77 @@
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+import { CopyObjectCommand, DeleteObjectCommand, GetObjectCommand, S3Client } from '@aws-sdk/client-s3';
+import csv from 'csv-parser';
+
+const BUCKET = 'cloudx-task5-import-service-data';
+
+const importFileParser = async (event: any) => {
+  let statusCode = 202;
+  console.log('[importFileParser] Starting...', { event });
+
+  try {
+    const s3 = new S3Client({ region: 'us-east-1' });
+
+    for (const record of event.Records) {
+      const getObjCmd = new GetObjectCommand({
+        Bucket: BUCKET,
+        Key: record.s3.object.key,
+      });
+
+      const s3Data = await s3.send(getObjCmd);
+
+      await processS3Data(s3, s3Data.Body, record.s3.object.key);
+    }
+  } catch (error) {
+    console.error(error);
+    statusCode = error.statusCode || 500;
+  }
+
+  console.log('All Done!');
+  return formatJSONResponse({
+    statusCode,
```

Comment:
Based on the `import-service/src/libs/api-gateway.ts`, this `statusCode` is under the `body` key, and you don't change the lambda response statusCode. It's always 200.

Reply:
Thanks, I didn't notice this.